src/libs/runtime-utils.js

```diff
@@ -167,7 +167,7 @@ export const machineCost = machineType => {
   return _.find(knownMachineType => knownMachineType.name === machineType, machineTypes).price
 }
 
-export const currentApp = _.flow(trimAppsOldestFirst, _.last)
+export const getCurrentApp = _.flow(trimAppsOldestFirst, _.last)
```

Comment:
functions are verbs variables are nouns, otherwise the variable that holds the result has to be named weird.